app/services/vot_parser.rb

```diff
@@ -0,0 +1,118 @@
+class VotParser
+  class ParseException < StandardError; end
+
+  ComponentValue = Data.define(:name, :description, :implied_component_values, :requirements)
+
+  SUPPORTED_COMPONENT_VALUES = [
+    ComponentValue.new(
+      name: 'C1',
+      description: 'Multi-factor authentication',
+      implied_component_values: [],
+      requirements: [],
+    ),
+    ComponentValue.new(
+      name: 'C2',
+      description: 'AAL2 conformant features are engaged',
+      implied_component_values: ['C1'],
+      requirements: [:aal2],
+    ),
+    ComponentValue.new(
+      name: 'Ca',
+      description: 'A phishing resistant authenticator is required',
+      implied_component_values: ['C1'],
+      requirements: [:phishing_resistant],
+    ),
+    ComponentValue.new(
+      name: 'Cb',
+      description: 'A PIV/CAC card is required',
+      implied_component_values: ['C1'],
+      requirements: [:hspd12],
+    ),
+    ComponentValue.new(
+      name: 'P1',
+      description: 'Identity proofing is performed',
+      implied_component_values: ['C2'],
+      requirements: [:identity_proofing],
+    ),
+    ComponentValue.new(
+      name: 'Pb',
+      description: 'A biometric comparison is required as part of identity proofing',
+      implied_component_values: ['P1'],
+      requirements: [:biometric_comparison],
+    ),
+  ].freeze
+
+  SUPPORTED_COMPONENT_VALUES_MAP = SUPPORTED_COMPONENT_VALUES.index_by(&:name).freeze
+
+  SUPPORTED_COMPONENT_VALUE_REQUIREMENTS = [
+    :aal2,
+    :phishing_resistant,
+    :hspd12,
+    :identity_proofing,
+    :biometric_comparison,
+  ].freeze
+  Result = Data.define(:vector_of_trust, *SUPPORTED_COMPONENT_VALUE_REQUIREMENTS)
```

Comment:
```suggestion
  SUPPORTED_COMPONENT_VALUE_REQUIREMENTS = [
    :aal2,
    :phishing_resistant,
    :hspd12,
    :identity_proofing,
    :biometric_comparison,
  ].freeze
  Result = Data.define(:vector_of_trust, *SUPPORTED_COMPONENT_VALUE_REQUIREMENTS) do
    alias_method :aal2?, :aal
    alias_method :phishing_resistant?, :phishing_resistant
    alias_method :hspd12?, :hspd12
    alias_method :identity_proofing?, :identity_proofing
    alias_method :biometric_comparison?, :biometric_comparison
  end
```
also that constant `SUPPORTED_COMPONENT_VALUE_REQUIREMENTS` is only used once so I'd even consider dropping it?